<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bookRetrival</class>
 <widget class="QWidget" name="bookRetrival">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>417</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>652</width>
    <height>417</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-20</y>
     <width>121</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:700; color:#8e4b5a;&quot;&gt;图书检索&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QLineEdit" name="queryword">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(171, 149, 152);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QComboBox" name="searchindex">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(114, 42, 57);</string>
      </property>
      <item>
       <property name="text">
        <string>检索字段</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>书名</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>作者</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>出版社</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>出版年份</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>学科</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="resulttable">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>591</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 100);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <column>
    <property name="text">
     <string>ISBN</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>书名</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>类别</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>作者</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>出版社</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>语言</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>出版年份</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>总库存量</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>价格</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>当前数量</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="retirval1">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>20</y>
     <width>50</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(116, 178, 65);</string>
   </property>
   <property name="text">
    <string>精确检索</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="retrival2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>20</y>
     <width>50</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(134, 160, 113);</string>
   </property>
   <property name="text">
    <string>模糊检索</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>370</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(55, 16, 24);</string>
   </property>
   <property name="text">
    <string>我要借书</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>471</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="2" column="3">
     <widget class="QComboBox" name="category">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(62, 114, 31);</string>
      </property>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>工学</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>理学</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>文学</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>医学</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>农学</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>交叉学科</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>经济学</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>管理学</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>教育学</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>哲学</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>法学</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>艺术学</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>历史学</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_5">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>作者</string>
      </property>
     </widget>
    </item>
    <item row="2" column="6">
     <widget class="QLineEdit" name="low_year">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(62, 114, 31);</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QLabel" name="label_6">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>年份</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="label_8">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>价格</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>出版社</string>
      </property>
     </widget>
    </item>
    <item row="2" column="7">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="2" column="8">
     <widget class="QLineEdit" name="high_year">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(62, 114, 31);</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="bookname">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(62, 114, 31);</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>类别</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="author">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(62, 114, 31);</string>
      </property>
     </widget>
    </item>
    <item row="0" column="8">
     <widget class="QLineEdit" name="high_price">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(62, 114, 31);</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QComboBox" name="press">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(62, 114, 31);</string>
      </property>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>电子工业出版社</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>人民邮电出版社</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>机械工业出版社</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>其他</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>书名</string>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QLineEdit" name="low_price">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(62, 114, 31);</string>
      </property>
     </widget>
    </item>
    <item row="0" column="7">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>--</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="search">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>340</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(122, 55, 69);</string>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clear">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>340</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(122, 55, 69);</string>
   </property>
   <property name="text">
    <string>清除</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>671</width>
     <height>411</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/library5.jpg);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <zorder>frame_3</zorder>
  <zorder>label</zorder>
  <zorder>frame</zorder>
  <zorder>resulttable</zorder>
  <zorder>retirval1</zorder>
  <zorder>retrival2</zorder>
  <zorder>pushButton</zorder>
  <zorder>frame_2</zorder>
  <zorder>search</zorder>
  <zorder>clear</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
